airbyte-bootloader/src/test/java/io/airbyte/bootloader/BootloaderAppTest.java

```diff
@@ -138,7 +138,7 @@ void testBootloaderAppBlankDb() throws Exception {
       val configsMigrator = new ConfigsDatabaseMigrator(configDatabase, configsFlyway);
       // this line should change with every new migration
       // to show that you meant to make a new migration to the prod database
-      assertEquals("0.40.11.001", configsMigrator.getLatestMigration().getVersion().getVersion());
+      assertEquals("0.40.11.002", configsMigrator.getLatestMigration().getVersion().getVersion());
```

Comment:
I don't think it matters but it should be 0.40.13.000

Reply:
I think there was an upgrade since Wednesday when I wrote this